### PR TITLE
Dev - Create mongo db models

### PR DESCRIPTION
**Pull request by jangap411**
Added basic mongodb models to the project

### CODE REVIEW

**Comment by jangap411** [comment]
Add model validations later

File: backend/models/Profile.js
Comment (digitalKuMuL) [inline backend/models/Profile.js:1]: Add validation later.

File: backend/models/Invoice.js
Comment (digitalKuMuL) [inline backend/models/Invoice.js:1]: basic validation

File: backend/models/Profile.js
Comment (jangap411) [reply backend/models/Profile.js:1]: noted

File: backend/models/Client.js
Comment (digitalKuMuL) [inline backend/models/Client.js:1]: basic validation

File: backend/index.js
Comment (digitalKuMuL) [inline backend/index.js:1]: basic validation

**Review by digitalKuMuL — APPROVED** [review]
basic validation. 
Please add validation on the next PR

**Comment by jangap411** [comment]
merging now ?

**Pull request merged**